doc/sphinx/source/conf.py

```diff
@@ -30,4 +30,4 @@
 html_static_path = ['_static']
 
 # -- Breathe configuration ---------------------------------------------------
-breathe_default_project = "parameters"
+breathe_default_project = "modelfit_parameters"
```

Comment:
Why duplicate information?
```suggestion
breathe_default_project = project
```